src/main/java/frc/robot/commands/swerve/SetSwerveDrive.java

```diff
@@ -37,6 +37,8 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+
+    // TODO: If you change value in here, change value in ShootNStrafe
```

Comment:
Is there a way we can make both change if one changes? In case someone doesn't change both variables?

Reply:
will work on soon, this is primarily so we can test the concept incase something else braakes :(